src/main/java/org/springframework/session/data/mongo/MongoOperationsSessionRepository.java

```diff
@@ -73,12 +73,12 @@
 
 	private Integer maxInactiveIntervalInSeconds = DEFAULT_INACTIVE_INTERVAL;
 	private String collectionName = DEFAULT_COLLECTION_NAME;
-	private AbstractMongoSessionConverter mongoSessionConverter = new JdkMongoSessionConverter(
-		Duration.ofSeconds(this.maxInactiveIntervalInSeconds));
+	private AbstractMongoSessionConverter mongoSessionConverter;
 	private ApplicationEventPublisher eventPublisher;
 
-	public MongoOperationsSessionRepository(MongoOperations mongoOperations) {
+	public MongoOperationsSessionRepository(MongoOperations mongoOperations, ClassLoader classLoader) {
```

Comment:
This would be a breaking change. Couldn't `MongoHttpSessionConfiguration` be changed to reuse the existing `mongoSessionConverter` setter?